classes/gateways/class.pmprogateway_stripe.php

```diff
@@ -3455,22 +3495,34 @@ static function using_legacy_keys() {
 		return $r;
 	}
 
-	static function has_connect_credentials() {
-		return ( 
-			pmpro_getOption( 'stripe_connect_user_id' ) &&
-			pmpro_getOption( 'live_stripe_connect_secretkey' ) &&
-			pmpro_getOption( 'test_stripe_connect_secretkey' ) &&
-			pmpro_getOption( 'live_stripe_connect_publishablekey' ) &&
-			pmpro_getOption( 'test_stripe_connect_publishablekey' )
-		);
+	static function has_connect_credentials( $gateway_environment = null ) {
```

Comment:
I like that this function has this parameter passed into it, other functions could use this capability to help with future unit testing.